recursio-client/UI/Menus/GameRooms/GameRoomSearch.gd

```diff
@@ -0,0 +1,77 @@
+extends PanelContainer
+class_name GameRoomSearch
+
+signal btn_create_game_room_pressed()
+signal btn_back_pressed()
+signal btn_join_game_room_pressed()
+
+
+onready var _room_filter: LineEdit = get_node("Content/TopBar/LineEdit_GameRoomFilter")
+onready var _game_room_list: ItemList = get_node("Content/ItemList_GameRooms")
+
+onready var _btn_create_room: Button = get_node("Content/TopBar/Btn_CreateGameRoom")
+onready var _btn_refresh_game_rooms: Button = get_node("Content/TopBar/Btn_Refresh")
+onready var _btn_back: Button = get_node("Content/TopBar/Btn_Back")
+onready var _btn_join_game_room: Button = get_node("Content/BottomBar/Btn_JoinGameRoom")
+
+# Game Room ID - Game Room Name
+var _game_room_dic := {}
+
+
+func _ready():	
+	_room_filter.connect("text_changed", self, "_on_filter_text_changed")
```

Comment:
Add `var _error =... `to supress warning for every connect.

Reply:
Done